### PR TITLE
Milestone 3: Desktop Version

### PR DESCRIPTION
**Pull request by CarlosIgreda**
## Portfolio Milestone 3: Desktop Version 

In this milestone I:

•	Created a new branch according to the GitHub flow rules
•	Added a .gitignore file
•	Set up the linters config
•	Used grid layout
•	Used flexbox
•	Created hover effect
•	Used button interactions
•	Created a UI
•	Used images and backgrounds to enhance the look of the website
•	Created forms with HTML5 validations
•	Used field validations
•	Collected data by using the Formspree service
•	Applied media queries

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,176 +68,280 @@ <h1 id="idh1">
...
-                        <div id="idlanguage" class="more-info">
-                            <h2>Languages</h2>
-                            <img src="./images/down-arrow.svg" alt="down-arrow">
+        <section class="aboutcontact">
+            <article class="aboutmyself">
+                <div class="gabout">
+                    <div id="div1about">
+                        <h2>About <br> Myself</h2>
+                        <p>
+                            Hello I’m a software developer! I can help you build a product , feature or website Look
```
Comment (MeqdamAlqudah) [inline index.html:283]: - [x] Please fix the overflow in all your web pages, if the user goes to the right then everything will go messy you should consider adding an overflow-x: hidden property to your body tag then you can realign all the items so everything looks like the design and responsive as we want.
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @CarlosIgreda , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Highlights
:+1: no linter errors 
:+1: your design looks like Figmas

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- fixing your page overflow 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (CarlosIgreda) [reply index.html:283]: Thanks for your review @MeqdamAlqudah. I've followed your suggestion and I added an overflow-x: hidden in the body tag and also now is responsive in the desktop version as you mentioned.

Comment (CarlosIgreda) [reply index.html:283]: Please @MeqdamAlqudah, could you check my pull request before sending the code review? Just to ensure that everything is okay. Thanks in advance.

**Review by Owusu-Desmond — APPROVED** [review]
Hi @CarlosIgreda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CarlosIgreda** [comment]
Thanks a lot for your code review @Owusu-Desmond, I'm doing my best !! 💪

**Pull request merged**